giraffe/src/utils/getMargins.ts

```diff
@@ -13,16 +13,24 @@ export const getMargins = (
   yAxisLabel: string,
   yTicks: number[],
   yTickFormatter: (tick: number) => string,
-  tickFont: string
+  tickFont: string,
+  yColumnLabels: string[]
```

Comment:
Why not use yTicks to store yColumnLabels and have yTicks be of type `number[]` for non-mosaic layers, and `string[]` for mosaic layers?

Reply:
and then yTickFormatter could be the identity function and be defined as type : `(tick: number| string)=> string`